apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/PolicyTemplates.tsx

```diff
@@ -26,7 +30,13 @@ export const PolicyTemplates = ({
   onSelectTemplate,
 }: PolicyTemplatesProps) => {
   const [search, setSearch] = useState('')
-  const templates = getGeneralPolicyTemplates(schema, table.length > 0 ? table : 'table_name')
+
+  let templates: PolicyTemplate[] = []
```

Comment:
this one's just a preference for immutability, prefer:
`const templates = schema === 'realtime' ? getRealtimePolicyTemplates() : getGeneralPolicyTemplates()`

Reply:
Fixed.